tools.go

```diff
@@ -0,0 +1,15 @@
+// +build tools
+
+package tools
```

Comment:
Hmm what is this file for?

Reply:
It was created by the operator-sdk. Since it's empty I'll delete it.

Reply:
Actually, I was thinking of a different file. It was created by the operator-sdk. It just imports packages and I don't really know what it's for.

Reply:
I believe this is only used for stuff that is built using "go modules". Cf. https://github.com/golang/go/issues/25922

We haven't switched to using modules yet, but Tom K. did take a whack at allowing us to maintain the necessary metadata (go.mod/sum) in the original KubeDirector code. So perhaps we should leave this file here as well. I do wonder why the SDK framework generated it... maybe it was left over from a previous try at making the operator skeleton using modules rather than dep?